### DISCUSSION TITLE
❗ Add Skeleton/ Placeholders to Orbit.

### DISCUSSION THREAD

**Discussion by alexasselin008** [gsoft-inc/sg-orbit#1118] [New components Ideas]
## Description

I understand that this request must come from a stakeholder, but i would like to open a discussion and have a written trace of this request.

In Sharegate Apricot, we use Skeleton/Placeholder to display the loading state of some components.
![placeholder](https://user-images.githubusercontent.com/38871812/116115182-20bdba00-a688-11eb-976b-9668bf798d0f.gif)

I believe a control like this could be added to Orbit in order to standardize our design.

Before coding any of those components, the first step would be to document When and Where to use placeholders/skeletons

Example in Chakra : https://chakra-ui.com/docs/feedback/skeleton

**Comment by GateNk** [comment]
@fraincs I put this in the Development project. Let me know if this should be an umbrella and if there is anything that I can do. Alex told me that this could be solved through a simple CSS anim.

**Comment by patricklafrance** [comment]
@GateNk I removed this request from the development backlog for now since there a still a few discussions which need to happens before we consider adding skeletons to Orbit.

**Comment by patricklafrance** [comment]
Skeleton has been in discussion for a while but it never seem to take off. Should this issue be deleted?
